### PR TITLE
Restructure vs4 mac

### PR DESCRIPTION
**Pull request by NicoVermeir**
### Description:

This change is the VS4Mac part of the project restructuring proposed in #254. Still to do:

Update release pipelines
Part of #254

### Checklist:
* [ ] I have commented my code, particularly in hard-to-understand areas
* [X] My changes generate no new warnings
* [ ] I have added tests that prove my fix is effective or that my feature works
* [X] New and existing unit tests pass locally with my changes
* [ ] I have tested my changes by running the extension in VS2017
* [ ] I have tested my changes by running the extension in VS2019
* [X] I have tested my changes by running the extension in VS4Mac
* [ ] If changes to the [documentation](https://github.com/Xavalon/XamlStyler/wiki) are needed, I have noted this in the description above

### CODE REVIEW

File: src/XamlStyler.Extension.Mac/gtk-gui/gui.stetic [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <target-gtk-version>2.12</target-gtk-version>
   </configuration>
   <import>
-    <widget-library name="../bin/Release/XamlStyler.Mac.dll" internal="true" />
+    <widget-library name="../bin/Debug/XamlStyler.Mac.dll" internal="true" />
```
Comment (grochocki) [inline src/XamlStyler.Extension.Mac/gtk-gui/gui.stetic:8]: Intentional?

File: src/XamlStyler/DocumentManipulation/NodeCollection.cs [hunk 2]
```diff
@@ -1,9 +1,10 @@
-// � Xavalon. All rights reserved.
+// ? Xavalon. All rights reserved.
```
Comment (grochocki) [inline src/XamlStyler/DocumentManipulation/NodeCollection.cs:1]: Something weird has been going on with this.

File: src/XamlStyler/MarkupExtensions/Parser/MarkupExtensionParser.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
-// � Xavalon. All rights reserved.
+// ? Xavalon. All rights reserved.
```
Comment (grochocki) [inline src/XamlStyler/MarkupExtensions/Parser/MarkupExtensionParser.cs:1]: Same

File: src/XamlStyler.Extension.Mac/gtk-gui/gui.stetic (hunk 1 above)
Comment (NicoVermeir) [reply src/XamlStyler.Extension.Mac/gtk-gui/gui.stetic]: no :)

File: src/XamlStyler/MarkupExtensions/Parser/MarkupExtensionParser.cs (hunk 3 above)
Comment (NicoVermeir) [reply src/XamlStyler/MarkupExtensions/Parser/MarkupExtensionParser.cs]: should be all fixed in new commit

File: src/XamlStyler/DocumentManipulation/NodeCollection.cs (hunk 2 above)
Comment (NicoVermeir) [reply src/XamlStyler/DocumentManipulation/NodeCollection.cs]: should be all fixed in new commit

**Pull request merged**